src/libshadertrap/src/executor.cc

```diff
@@ -66,7 +67,7 @@ template <typename T>
 void DumpFormatEntry(const char* data,
                      const CommandDumpBufferText::FormatEntry& format_entry,
                      std::ofstream* text_file, size_t* index) {
-  std::vector<int32_t> values(format_entry.count);
+  std::vector<T> values(format_entry.count);
```

Comment:
Hmm, this change was already merged before you started working on your PR, which suggests you didn't make your PR from an up to date main. It's always best to start with an up to date main before you do a branch for a new feature.